Changelog.md

```diff
@@ -17,6 +17,7 @@ Bugfixes:
  * Assembly output: Do not mix in/out jump annotations with arguments.
  * Code Generator: Annotate jump from calldata decoder to function as "jump in".
  * Type Checker: Properly detect different return types when overriding an external interface function with a public contract function.
+ * Optimizer: Fix nondeterminism bug related to the boost version and constants representation.
```

Comment:
Please highlight here your comment:
> This bug cannot have resulted in incorrect code because we have a run-time check that the computed value is correct.